docker/nginx/mime.types

```diff
@@ -0,0 +1,50 @@
+types {
```

Comment:
This seems like mostly the default config? In that case you don't need to specify it.

Reply:
I believe the only types we have are images and pdf? Both which are included in the default config already.
Also zf should take care of setting the mime type for everything served through there.

Reply:
It is generally better to only include settings in Nginx config which you are overriding. Then there is no confusion about what’s default and what not. This also makes upgrades a lot easier.

Reply:
SVG was not included to give one example